source/index.md

```diff
@@ -44,17 +44,35 @@ curl "https://secure.ombushop.com/api/products?secret=abcdef1234567890"
 [
   {
     "id": 120,
-    "name": "Zapatillas",
-    "description": "<p>Zapatillas de ultima moda</p>",
+    "name": "Botas Leñador",
+    "description": "Confeccionadas en gamuza",
     "created_at": "2015-06-25 14:43:51 -0300",
     "updated_at": "2015-07-03 17:17:40 -0300",
-    "permalink": "zapatillas",
+    "permalink": "botas-lenador",
     "deleted_at": null,
     "meta_description": "Descripción",
     "meta_keywords": "Palabras clave",
     "count_on_hand": 10,
     "seller_id": 24,
-    "currency": "ARS"
+    "currency": "ARS",
+    "images": [
+      {
+        "id": 15877,
+        "viewable_id": 120,
+        "viewable_type": "Product",
+        "attachment_content_type": "image/jpg",
+        "attachment_file_name": "0.jpg",
+        "position": 1,
+        "attachment_width": 836,
+        "attachment_height": 494,
+        "image_paths": [
```

Comment:
@mauro-oto Sorry, I forgot to see this in https://github.com/ombulabs/ombushop-api/pull/15 but could you rename this key as "urls" instead of "image_paths"